src/affine.jl

```diff
@@ -46,16 +47,18 @@ function qd_affine_coarse(fixed, moving, mxshift, linmins, linmaxs, SD;
     params = position(box, x0)
     tfmcoarse0 = linmap(params, moving, SD, initial_tfm)
     best_shft, mm = best_shift(fixed, moving, mxshift, thresh; normalization=:intensity, initial_tfm=tfmcoarse0)
```

Comment:
If you get rid of `SD` as an argument to `linmap`, you'll want an `arrayscale` just before this line.